src/GraphBuilder.js

```diff
@@ -107,6 +104,18 @@ export default class GraphBuilder {
     }
   }
 
+  /**
+   * Returns a literal with an optional language tag
+   * @param {Object} item from the redux store
+   * @return {rdf.LiteralExt} the literal with a language value
+   */
+  createLiteral(item) {
+    if (item.lang) {
```

Comment:
If `item.lang` is truthy, is it a safe bet that it has an `items` array of length > 0 that contains things with an `id`? If so, this is fine. If we're less sure, we can do something like this and ditch the condition:

```javascript
return rdf.literal(item.content, item?.lang?.items?.[0]?.id)
```

Reply:
Yes, we're sure of the shape.   But I did revise this method to use the safe navigation.